Workspaces/Common/Scripts/WorkspaceButton.cs

```diff
@@ -211,7 +208,8 @@ void Awake()
 			if (m_SwapIconsOnClick && m_AlternateIconSprite)
```

Comment:
the if(m_Button) check needs to surround this whole block or the next line will have a nullref